designer/server/views/help/accessibility-statement.html

```diff
@@ -14,12 +14,12 @@ <h1 class="govuk-heading-xl">Accessibility Statement</h1>
 
                     <p class="govuk-body">This statement applies to the Digital Form Designer website.</p>
 
-                    <p class="govuk-body">This website is run by Home Office Digital, Data and Technology service. 
+                    <p class="govuk-body">This website is run by Home Office Digital, Data and Technology service.
```

Comment:
This will eventually need parameterised, but it's fine atm since our version has ipsum anyways: https://forms.test.gids.dev/help/accessibility-statement (dont think we link to it anywhere)